runtime/criusupport/criusupport.cpp

```diff
@@ -544,6 +546,9 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 		malloc_trim(0);
 		VM_VMHelpers::setVMState(currentThread, J9VMSTATE_CRIU_SUPPORT_CHECKPOINT_PHASE_END);
 		systemReturnCode = criu_dump();
+
+		restoreStart = j9time_current_time_nanos(&timeSuccess);
```

Comment:
I think this can use or move existing `restoreNanoUTCTime = j9time_current_time_nanos(&success)`.